scapy/arch/windows/__init__.py

```diff
@@ -539,7 +540,7 @@ def sndrcv(pks, pkt, timeout = 2, inter = 0, verbose=None, chainCC=0, retry=0, m
                                     if r.answers(sentpkt):
                                         ans.append((sentpkt, r))
                                         if verbose > 1:
-                                            os.write(1, "*")
+                                            sys.stdout.write("*")
```

Comment:
ok put it back or add a flush, I changed this because it won't be "printed" on code that wraps sys.stdout and not os.write